tox.ini

```diff
@@ -175,3 +175,43 @@ deps =
 passenv = *
 commands =
     python {env:DRIVERS_TOOLS}/.evergreen/auth_aws/setup_secrets.py {posargs}
+
+[testenv:setup-encryption]
+deps =
+    {[testenv:aws-secrets]deps}
+    pykmip
+passenv = *
+allowlist_externals =
+    bash
+commands =
+    {[testenv:aws-secrets]commands} drivers/csfle
+    bash ./.evergreen/start-kmip-server.sh
+
+[testenv:teardown-encryption]
+passenv = *
+allowlist_externals =
+    bash
+    rm
+commands =
+    bash ./.evergreen/stop-kmip-server.sh
+    rm -rf libmongocrypt*
+
+[testenv:test-encryption]
+deps =
+    {[testenv:test-eg]deps}
+    {[testenv:setup-encryption]deps}
+extras =
+    {[testenv:test-eg]extras}
+allowlist_externals =
+    {[testenv:test-eg]allowlist_externals}
+    {[testenv:teardown-encryption]allowlist_externals}
+passenv = *
+setenv =
+    LIBMONGOCRYPT_URL = https://s3.amazonaws.com/mciuploads/libmongocrypt/macos/master/latest/libmongocrypt.tar.gz
```

Comment:
This would be the first time we hard code macos into tox. Can we avoid this?

Reply:
I think the best move is to just remove this tox env entirely, as we need a different `LIBMONGOCRYPT_URL` for each platform.